8Knot/assets/cursor-fix.css

```diff
@@ -0,0 +1,39 @@
+/* ==== TEXT CURSOR VISIBILITY ==== */
+
+.searchbar-dropdown input::placeholder {
+    background-color: var(--placeholder-bg-color)!important;
+    color: var(--placeholder-text-color);
+    font-size: var(--placeholder-font-size);
+    line-height: var(--placeholder-line-height);
+    position: static !important;
+    top: auto !important;
+}
+
+.searchbar-dropdown input:focus::placeholder {
+    background-color: var(--placeholder-bg-color)!important;
+    color: var(--placeholder-text-color);
+    font-size: var(--placeholder-font-size);
+    line-height: var(--placeholder-line-height);
+    position: static !important;
+    top: auto !important;
+}
+
+.mantine-MultiSelect-input::placeholder {
+    background-color: var(--placeholder-bg-color);
+    color: var(--placeholder-text-color);
+    font-size: var(--placeholder-font-size);
+    line-height: var(--placeholder-line-height);
+    position: static !important;
+    top: auto !important;
+}
+
+
+/* Ensure text cursor is visible with white color */
+
+.mantine-MultiSelect-input {
+    caret-color: #ffffff !important;
+}
```

Comment:
IMO this is entirely too much CSS to fix one issue.

I have also seen a lot of very similar CSS in other files